profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.module

```diff
@@ -252,15 +265,107 @@ function nexteuropa_multilingual_form_workbench_moderation_node_unpublish_form_a
+        }
+      }
+      // 2. Add new warning message to URL alias field.
+      $custom_alias_description = $form['path']['alias']['#description'];
+      $message = t('This URL alias will be used for ALL translations of this content.');
+      $message_attributes = array(
+        'class' => array('messages', 'warning'),
+        'role'  => array('alert'),
+        'id'    => array('warning_alias_message'),
+      );
+      $variables = array(
+        'message' => $message,
+        'message_attributes' => $message_attributes,
+      );
+      $custom_alias_description .= theme('nexteuropa_multilingual_message', $variables);
+      $form['path']['alias']['#description'] = $custom_alias_description;
+    }
+  }
+}
+
+/**
+ * Gets alias info for the entity and LANGUAGE_NONE langcode.
+ *
+ * @param object $entity
+ *   The entity for which data are needed.
+ * @param string $entity_type
+ *   The entity type.
+ *
+ * @return bool|array
+ *   FALSE if nothing is found; otherwise an associative array with
+ *   LANGUAGE_NONE data on the node alias.
+ *   The data contained in the associative array have the following keys:
+ *   - pid: The pid of the LANGUAGE_NONE alias of the node.
+ *   - alias: The the LANGUAGE_NONE alias.
+ *   - pathauto: The flag indicating if pathauto is activated for the node.
+ */
+function _nexteuropa_multilingual_get_und_entity_alias_info($entity, $entity_type) {
+  // This function only support entity translation for now.
+  list(, , $bundle) = entity_extract_ids($entity_type, $entity);
+  $pattern = pathauto_pattern_load_by_entity($entity_type, $bundle, LANGUAGE_NONE);
+  if (empty($pattern)) {
+    return FALSE;
+  }
+  // Get the corresponding LANGUAGE_NONE alias.
+  $entity_uri = $entity->path['source'];
+  $language_none_path = path_load(array('source' => $entity_uri, 'language' => LANGUAGE_NONE));
+  $language_none_alias = $language_none_path['alias'];
+
+  // Check if pathauto is activtedin LANGUAGE_NONE case.
+  // Logic is taken from Pathauto itself.
+  // @see pathauto_entity_presave() & pathauto_field_attach_form().
+  module_load_include('inc', 'pathauto');
+  $pathauto_alias = pathauto_create_alias($entity_type, 'return', $entity_uri, array($entity_type => $entity), $bundle, LANGUAGE_NONE);
+  $pathauto = ($language_none_alias != $entity_uri && $language_none_alias == $pathauto_alias);
+
+  if (isset($entity_uri->workbench_moderation) && isset($entity_uri->workbench_moderation['published'])) {
+    $pathauto = FALSE;
+  }
+
+  return array(
+    'pid' => $language_none_path['pid'],
+    'alias' => $language_none_alias,
+    'pathauto' => $pathauto,
+    'language' => $language_none_path['language'],
+  );
+}
+
+/**
  * Updates form with a warning message.
  */
 function _nexteuropa_multilingual_warning_message(&$form, $node) {
 
   $translatable = variable_get('language_content_type_' . $node->type, 0);
   if (!empty(variable_get('warning_message_languages', "")) && $translatable !== '0') {
```

Comment:
The sentence correction ("will be updated !" => "will be updated!");, I will put it in the install hook but not in the hook update because maybe somewhere the variable value has been changed.